README.md

```diff
@@ -1,20 +1,26 @@
-# Sample App Template
+<p>
+<a href="#start"><img height="30rem" src="https://raw.githubusercontent.com/arcana-network/branding/main/an_logo_light_temp.png"/></a>
+</p>
+<p>
+<a title="MIT License" href="https://github.com/arcana-network/license/blob/main/LICENSE.md"><img src="https://img.shields.io/badge/license-MIT-blue"/></a>
+<a title="Twitter" href="https://twitter.com/ArcanaNetwork"><img alt="Twitter URL" src="https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Ftwitter.com%2FArcanaNetwork"/></a>
+</p><p id="start" align="center">
+</p>
 
-This is a simple Vue application template. Use it as your starter kit to build a full-fledged dApp integrated with Arcana Auth and Storage SDKs.
+# Sample Demo App Template
 
-You can use this template to build a [Sample Demo App](https://github.com/arcana-network/demo-app) like application and learn how to enable user onboarding and data privacy features in dApp.
+This is a simple Vue application template. Use it as your starter kit to build a full-fledged dApp integrated with Arcana Auth and Storage SDKs.
 
-Refer to [Arcana Network documentation](https://docs.beta.arcana.network/docs/starter-app) for step by step instructions on how to use this template and integrate with Arcana Auth and Storage SDKs for enabling user onboarding and data privacy features.
+## ⚙️ Installation
 
-## Quick Start
+### Prerequisites
 
-- Prerequisites:
+- You'll need latest LTS versions of **`node.js`** and **`npm`** installed in your system for this code to work.
 
-  - You'll need latest LTS versions of **`node.js`** and **`npm`** installed in your system for this code to work.
-  - You'll need to register and configure your dApp using [Arcana's Developer Dashboard](https://dashboard.beta.arcana.network). Configure social authentication mechanism, storage region, storage limits and other settings using the dashboard.
+- You must register and configure your dApp using [Arcana's Developer Dashboard](https://dashboard.beta.arcana.network). Configure social authentication mechanism, storage region, storage limits and other settings using the dashboard.
```

Comment:
I think we should link to the main dashboard: https://dashboard.arcana.network. Right now, that redirects to the beta dashboard but this will change in the future.